addons/event_sale/models/event_registration.py

```diff
@@ -60,6 +74,19 @@ def _synchronize_so_line_values(self, so_line):
             }
         return {}
 
+    def _sale_order_ticket_type_change_notify(self, new_ticket_id):
+        for registration in self:
+            render_context = {
+                'registration': registration,
+                'old_ticket': registration.event_ticket_id.name,
+                'new_ticket': new_ticket_id.name
```

Comment:
old_ticket_name
new_ticket_name

To make it clear these are not browserecords.